resources/views/feedbackboards/include/feedback.blade.php

```diff
@@ -83,10 +86,11 @@
         <div class="text-muted text-end mt-2">
             <em>
                 <sub>
-                    @can('board')
-                        By {{ $feedback->user?->name ?? "before we kept track!" }}
-                    @endcan
-                    -- {{ $feedback->created_at->format("j M Y, H:i") }}
+                    @if(Auth::user()->can("board") || $feedback->category->show_publisher)
+                        By {{ $feedback->user?->name ?? "before we kept track!" }} --
```

Comment:
```suggestion
                       @if(!empty($feedback->user?->name))
                           By {{ $feedback->user->name }} --
                       @else
                           Before we kept track --
                       @endif
```

I think it's a little neater if you don't get 'By before we kept track!' but 'Before we kept track'
But that's just my opinion on it

Reply:
This is also what we do at the membercards so I just wanted to keep it consistent